br/pkg/task/restore.go

```diff
@@ -377,6 +377,16 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 		return nil
 	}
 
+	// create policy if backupMeta has policies.
+	policies, err := client.GetPlacementPolicies()
```

Comment:
I think we should check if it is full restore, and:

1. full restore. restore all policies got from met
2. otherwise. only restore policies from filtered tables and schemas

Reply:
okay. fixed.